sdk/core/azure-core/azure/core/tracing/_abstract_span.py

```diff
@@ -45,15 +45,15 @@ class SpanKind(Enum):
 class AbstractSpan(Protocol):
     """Wraps a span from a distributed tracing implementation."""
 
-    def __init__(self, span=None, name=None):  # pylint: disable=super-init-not-called
-        # type: (Optional[Any], Optional[str]) -> None
+    def __init__(self, span=None, name=None, **kwargs):  # pylint: disable=super-init-not-called
```

Comment:
This is a breaking change - although it is unlikely that anyone has actually created custom span types...

Reply:
how is adding kwargs breaking?